src/buildifier/buildifier.ts

```diff
@@ -37,18 +37,18 @@ export type BuildifierFileType = "build" | "bzl" | "workspace" | "default";
  *
  * @param fileContent The BUILD or .bzl file content to process, which is sent
  * via stdin.
- * @param type Indicates whether to treat the file content as a BUILD file or a
- * .bzl file.
+ * @param filePath The path to the file being formatted, relative to the
+ * workspace root.
  * @param applyLintFixes If true, lint warnings with automatic fixes will be
  * fixed as well.
  * @returns The formatted file content.
  */
 export async function buildifierFormat(
   fileContent: string,
-  type: BuildifierFileType,
+  filePath: string,
   applyLintFixes: boolean,
 ): Promise<string> {
-  const args = [`--mode=fix`, `--type=${type}`];
+  const args = [`--mode=fix`, `--path=${filePath}`];
```

Comment:
The docs state that this argument has only one leading dash. https://github.com/bazelbuild/buildtools/blob/main/buildifier/buildifier.go#L66-L70
But it seems to accept both.